### PR TITLE
Fix: Added Contribution Section to Readme

### LINKED ISSUE #545: Feature: Adding contributors section to the README.md file.
There is no Contributors section in readme file .
As we know Contributions are what make the open-source community such an amazing place to learn, inspire, and create.
The Contributors section in a README.md file is important as it acknowledges and gives credit to those who have contributed to a project, fosters community and collaboration, adds transparency and accountability, and helps document the project's history for current and future maintainers. It also serves as a form of recognition, motivating contributors to continue their efforts.
contributors

### PR DESCRIPTION
**Pull request by Kalyanimhala**
Fixes #545
Successfully added Contributions section to the Readme.md.
@akrambek @jfallows  review it and merged the PR. Thank You!

### CODE REVIEW

**Comment by Kalyanimhala** [comment]
@jfallows Please review my PR!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Contributor Covenant Code of Conduct
```
Comment (jfallows) [inline CODE_OF_CONDUCT.md:1]: We already have a CODE_OF_CONDUCT.md at the standard location.
Please see https://github.com/aklivity/zilla/blob/develop/.github/CODE_OF_CONDUCT.md.

File: README.md [hunk 2]
```diff
@@ -174,6 +174,11 @@ Please review the [Zilla Roadamp][zilla-roadmap]. If you have a request or feedb
 Looking to contribute to Zilla? Check out the [Contributing to Zilla](./.github/CONTRIBUTING.md) guide.
 ✨We value all contributions, whether it is source code, documentation, bug reports, feature requests or feedback!
 
+### All Thanks To Our Contributors:
+<a href="https://github.com/jfallows/zilla/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=jfallows/zilla" />
+</a>
```
Comment (jfallows) [inline README.md:180]: The repository should be `aklivity/zilla` instead of `jfallows/zilla`.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -174,6 +174,11 @@ Please review the [Zilla Roadamp][zilla-roadmap]. If you have a request or feedb
 Looking to contribute to Zilla? Check out the [Contributing to Zilla](./.github/CONTRIBUTING.md) guide.
 ✨We value all contributions, whether it is source code, documentation, bug reports, feature requests or feedback!
 
+### All Thanks To Our Contributors:
```
Comment (jfallows) [inline README.md:177]:
```suggestion
### Many thanks to our contributors!
```

**Review by jfallows — CHANGES_REQUESTED** [review]

**Review by jfallows — APPROVED** [review]

**Review by jfallows — APPROVED** [review]

**Pull request merged**